docs/content/integrations/dbt/using-dbt-with-dagster-plus.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: "Using dbt with Dagster+ | Dagster Docs"
+description: Deploy your dbt & Dagster project in Dagster+.
+---
+
+# Using dbt with Dagster+
+
+Importing an existing dbt project in Dagster+ allows you to automatically load your dbt models as Dagster assets. This can be be done with:
+
+- An existing dbt project that is not already using Dagster, or
+- A Dagster project in which your dbt project is included
+
+In this guide, we'll demonstrate by using an existing dbt project that doesn't use Dagster.
+
+---
+
+## Prerequisites
+
+To follow the steps in this guide, you'll need **Dagster+ Organization Admin**, **Admin**, or **Editor** permissions. This is required to create a code location.
+
+You'll also need **an existing dbt project** that contains the following files in the repository root:
+
+- [`dbt_project.yml`](https://docs.getdbt.com/reference/dbt_project.yml)
+- [`profiles.yml`](https://docs.getdbt.com/docs/core/connect-data-platform/profiles.yml)
+
+---
+
+## Step 1: Import your project in Dagster+
+
+<Note>
+  <strong>Heads up!</strong> This guide focuses on Serverless deployments in
+  Dagster+, but dbt projects may also be deployed if using a Hybrid deployment.
+  For more information about Hybrid deployments, refer to the{" "}
+  <a href="/dagster-plus/deployment/hybrid#hybrid-deployments-in-dagster">
+    Hybrid deployment documentation
+  </a>
+  .
+</Note>
+
+In this section, we'll demonstrate how to import an existing project Dagster+. Our example imports the project from a GitHub repository, but Dagster+ also supports Gitlab.
+
+1. Sign in to your Dagster+ account.
+
+2. Navigate to **Deployment > Code locations**.
+
+3. Click **Add code location**.
+
+4. Click **Import a dbt project**, then **Continue**.
+
+5. At this point, you'll be prompted to select either GitHub or Gitlab. For this guide, we'll select **GitHub**.
+
+6. You'll be redirected to GitHub, where you'll be prompted to authorize the Dagster+ application. **Note**: The profile or organization you're using to authorize Dagster+ must have read and write access to the repository containing the project.
+
+   After the authorization is complete, you'll be redirected back to Dagster+.
+
+7. In Dagster+, locate and select the repository containing the project by using the dropdowns. **Note**: dbt projects must have `dbt_profiles.yml` and `profiles.yml` files in the repository root or an error will display.
+
+8. Click **Continue**. Dagster+ will begin importing the project and open a pull request to update the repository. This pull request adds a few files, which we'll discuss in the next section.
```

Comment:
This is true only for dbt-only projects - for dbt & Dagster projects, the files are directly committed to the default branch. Maybe a note about that would be good?

Reply:
Definitely, thanks for calling this out

Reply:
Updated!